ChakraCore.Debugger.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>ChakraCore.Debugger</id>
+    <version>$version$</version>
+    <description>Debugging companion library for the ChakraCore JavaScript engine.</description>
+    <authors>Microsoft</authors>
+    <projectUrl>https://github.com/Microsoft/ChakraCore-Debugger</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+  </metadata>
+
+  <files>
+    <file src="lib\Debugger.Protocol\**\*.h" target="include\Chakra\Debugger\Protocol" />
+    <file src="lib\Debugger.ProtocolHandler\**\*.h" target="include\Chakra\Debugger\ProtocolHandler" />
+    <file src="lib\Debugger.Service\**\*.h" target="include\Chakra\Debugger\Service" />
```

Comment:
We probably don't need all of these headers to end up in the package, the design was such that only `ChakraDebugService.h` and `ChakraDebugProtocolHandler.h` should actually be required by anyone embedding this code.

Reply:
Noted! I'll only export those two, and their transitive dependency headers.

Reply:
They should only ever depend on `ChakraCore.h`, so I believe those two represent the complete set of required headers.

Reply:
Will place them in the include path's root, then.